ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -60,6 +60,7 @@ Control {
     property bool overrideBackground: false
     property bool profileClickable: true
     property bool hideMessage: false
+    property bool isInPinnedPopup
```

Comment:
Isn't it dangerous without default value?

Reply:
`property bool` is `false` by default